include/revng/Model/model-schema.yml

```diff
@@ -189,6 +190,7 @@ definitions:
       - name: Index
         doc: The argument index.
         type: uint64_t
+        default: -1
```

Comment:
Worth it? What's you reasoning?

Reply:
OK, but what's the practical implication of leaving `default: 0`? :thinking:

Reply:
None. Except for the manual assert checking for this exact thing:
[`95ab93a` (#528)](https://github.com/revng/revng/pull/528/commits/95ab93a170e6a1897ea0447cb30245eebd7320b1#diff-9fc9a9c46221bd88ccba85350b3e949fd3d4e71c11c862b1bfa17d3e342c3997R87)

Reply:
I'm not sure about the benefit of this thing. I'm tempted to just drop it.

Your call.